packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/messages.g.m

```diff
@@ -4,7 +4,7 @@
 // Autogenerated from Pigeon (v18.0.0), do not edit directly.
 // See also: https://pub.dev/packages/pigeon
 
-#import "messages.g.h"
+#import "./include/video_player_avfoundation/messages.g.h"
```

Comment:
I changed this manually. This will get overwritten by pigeon until https://github.com/flutter/flutter/issues/147587 is fixed.